Sources/SwiftSentry/Options.swift

```diff
@@ -23,7 +23,7 @@ public final class Options {
   /// "`CFBundleIdentifier`@`CFBundleShortVersionString`+`CFBundleVersion`"
   ///
   /// If `customReleaseName` is set, that value will always be used instead.
-  public var releaseName: String? = {
+  public var releaseName: String? {
```

Comment:
This change semantics in that we can't assign to `releaseName` anymore, which we do in Arc. Why was `customReleaseName` needed when we could already assign to `releaseName` to override the default value?

Reply:
Yea no clue, I'm sure this all stemmed from me not correctly identifying the closure-based initializer pattern.